backend/app/comment_handler.go

```diff
@@ -0,0 +1,168 @@
+package app
+
+import (
+	"errors"
+	"log"
+	"net/http"
+
+	"gorm.io/gorm"
+
+	"github.com/codescalersinternships/Flyspray/models"
+	"github.com/gin-gonic/gin"
+)
+
+// Response is a struct that holds the ok response
+type Response struct {
+	Message string           `json:"message"`
+	Payload []models.Comment `json:"payload"`
+}
+
+// ErrorResponse is a struct that holds the error response
+type ErrorResponse struct {
+	Error string `json:"error"`
+}
+
+// CreateComment creates a comment on a specific bug
+func (app *App) CreateComment(c *gin.Context) {
+
+	comment := models.Comment{}
+
+	if err := c.ShouldBindJSON(&comment); err != nil {
+		log.Println("failed to create the comment:", err)
+		c.JSON(http.StatusBadRequest,
+			ErrorResponse{Error: "bugID,ownerID and summary of the comment should be provided "})
```

Comment:
bugID -> bug ID
ownerID -> owner ID
leave space after ,
remove the last space
errors and msgs should be user friendly as the user will deal with them, please check all